src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -85,6 +85,23 @@
 public interface GameRegistry {
 
     /**
+     * Creates a catalog key.
+     *
+     * @param namespace The namespace
+     * @param value The value
+     * @return A new catalog key
+     */
+    CatalogKey createKey(final String namespace, final String value);
```

Comment:
Frankly these methods offer little to nothing at all. CatalogKey has the create method (should be renamed of) which makes this method pointless and resolve can be put there as well. In what case would anyone ever reach out to the GameRegistry to create a CatalogKey...?